tests/test_display.py

```diff
@@ -81,6 +96,23 @@ def test_display_default(self, mock_put_text):
                                          position=(224, 294),
                                          thickness=self.thickness)
 
+    @patch('sense.display.DisplayClassnameOverlay._display_class_name')
+    def test_display_default_within_duration(self, mock_display_class_name):
+        test_display = base_display.DisplayClassnameOverlay(thresholds={'Dabbing': 0.5}, duration=.1)
+        start_time = time.perf_counter()
+        test_display.display(self.img, {'sorted_predictions': [['Dabbing', 0.6]]})
+
+        call_count = 0
+        while time.perf_counter() - start_time <= test_display.duration:
+            test_display.display(self.img, {'sorted_predictions': [['Dabbing', 0.6]]})
```

Comment:
The point here is that the inputs don't even need to contain a high prediction for that class and it will still be displayed